### DISCUSSION TITLE
FANUC2022 Programm Format

### DISCUSSION THREAD

**Discussion by Futoro** [iSorp/macro-executor#43] [Q&A]
Will the FANUC 2022 Program Format be added to the Macro Executor? There are a few differences compared to the older FANUC 2003 Program Format, which has been used until now. See here:
[FANUC 500i-A - Macro Compiler-Executor Programming Manual (B-64803EN-5) (1).pdf](https://github.com/user-attachments/files/16828432/FANUC.500i-A.-.Macro.Compiler-Executor.Programming.Manual.B-64803EN-5.1.pdf)

**Comment by iSorp** [comment]
Thanks for pointing that out.
It's not planned at the moment, but it's definitely something we need to look at.

**Reply by iSorp** [reply]
This will be implemented in 2026
